lib/docs/filters/pony/clean_html.rb

```diff
@@ -0,0 +1,9 @@
+module Docs
+  class Pony
+    class CleanHtmlFilter < Filter
+      def call
+        doc
+      end
+    end
+  end
+end
```

Comment:
If you don’t need to do anything here, you can remove this filter entirely.

Reply:
Ah, I was under the impression it was mandatory. I'll remove it.